packages/analytics-core/src/analytics.js

```diff
@@ -62,27 +54,71 @@ export default class AvAnalytics {
   }
 
   handleEvent = event => {
-    if (
-      isModifiedEvent(event) ||
-      (event.type === 'click' && !isLeftClickEvent(event)) ||
-      !isValidEventTypeOnTarget(event)
-    ) {
+    if (this.invalidEvent(event)) {
       return;
     }
+
     const target = event.target || event.srcElement;
-    const analyticAttrs = getAnalyticAttrs(target);
+    const { path } = event;
```

Comment:
`event.path` is non-standard (not event listed on https://developer.mozilla.org/en-US/docs/Web/API/Event). I know chrome has it, but firefox and probably others don't. You might want to try [composedPath](https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath) as a fallback. It is a standard, but still [not implemented everywhere](https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath#Browser_compatibility) such as IE and Edge.

Reply:
why can't everything just be available everywhere 😦

Reply:
You can always just recursively get the node's parent until you hit the body or null, that should work everywhere.

Reply:
Yeah that's what I am doing. Found this polyfill but just going to use the function instead of creating as a polyfill.

https://gist.github.com/kleinfreund/e9787d73776c0e3750dcfcdc89f100ec